app/settings/requires-elevation/admin/template.hbs

```diff
@@ -143,7 +143,7 @@
             </ul>
           {{else}}
             <button {{action 'showOtpRecoveryCodes'}} class="btn btn-primary" type="submit">
-              Show backup codes
+              Show Backup Codes
```

Comment:
Thanks - can you help me understand the difference between the first two cases there?